dev-python/pyproject-fmt-rust/pyproject-fmt-rust-1.0.6.ebuild

```diff
@@ -144,14 +144,21 @@ LICENSE+="
 "
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="+stable-api"
 
 distutils_enable_tests pytest
 
 QA_FLAGS_IGNORED="usr/lib/py.*/site-packages/pyproject_fmt_rust/_lib.*.so"
 
 src_prepare() {
 	sed -i -e '/strip/d' pyproject.toml || die
+	if ! use stable-api; then
+		sed -i -e 's:"abi3-py38"::' Cargo.toml || die
+	fi
+
 	distutils-r1_src_prepare
+
+	export UNSAFE_PYO3_SKIP_VERSION_CHECK=1
```

Comment:
What's this about?

Reply:
Disabling stable API enables Python compatibility check, so we need to allow it to use Python 3.13.